ts-tests/tests/test-evm-fill-block.ts

```diff
@@ -0,0 +1,132 @@
+import { expect } from "chai";
+import { step } from "mocha-steps";
+import { describeWithAcala } from "./util";
+import { BodhiSigner } from "@acala-network/bodhi";
+import { submitExtrinsic } from "./util";
+import { BigNumber } from "ethers";
+
+describeWithAcala("Acala RPC (EVM fill block)", (context) => {
+    let alice: BodhiSigner;
+    let alice_stash: BodhiSigner;
+
+    const FixedU128 = BigNumber.from('1000000000000000000');
+
+    before("init wallets", async function () {
+        [alice, alice_stash] = context.wallets;
+    });
+
+    step("evm create fill block", async function () {
+        /*
+        pragma solidity ^0.8.0;
+        contract Contract {}
+        */
+
+        const contract = "0x6080604052348015600f57600080fd5b50603f80601d6000396000f3fe6080604052600080fdfea2646970667358221220b9cbc7f3d9528c236f2c6bdf64e25ac8ca17489f9b4e91a6d92bea793883d5d764736f6c63430008020033";
+
+        const creates = Array(15).fill(context.provider.api.tx.evm.create(
+            contract,
+            0,
+            2_000_000,
+            100_000,
+            []
+        ));
+
+        const tx = context.provider.api.tx.utility.batchAll(creates);
```

Comment:
this is not the same as it was

Reply:
`test-service` can manually generate blocks, send all of the transactions to mempool, and then generate blocks, but mandala cannot, so I use batchAll to test

Reply:
can't we start manual-sealing, feed with tx and then build blocks and see how many tx are included per block?